src/utils.ts

```diff
@@ -376,7 +376,7 @@ export const removeLumigoFromStacktrace = (handleReturnValue) => {
     const { stack } = err;
     const stackArr = stack.split('\n');
 
-    const pattern = '/dist/lumigo.js:';
+    const pattern = 'lumigo';
```

Comment:
hmmm this is a bit problematic.
people tend to have test functions, so you see people with file name "lumigo_test", or handlers `lumigo_handler`.
Why do you change it?

Reply:
Agree, can you explain how it relates to the issue?

Reply:
The use throws an Object and we try to filter only `dist/lumigo.js`
but the user is has a tracer that is not packed to a single file thus the stack contains a different string pattern: 

`Error: [object Object]
    at Object.normalizeLambdaError (/opt/nodejs/node_modules/@lumigo/tracer/dist/tracer.js:210:15)
    at /opt/nodejs/node_modules/@lumigo/tracer/dist/tracer.js:304:54
    at step (/opt/nodejs/node_modules/@lumigo/tracer/dist/tracer.js:33:23)
    at Object.next (/opt/nodejs/node_modules/@lumigo/tracer/dist/tracer.js:14:53)
    at fulfilled (/opt/nodejs/node_modules/@lumigo/tracer/dist/tracer.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)`
    
    we should include this case aswell